.github/workflows/pr_tests.yml

```diff
@@ -149,6 +148,7 @@ jobs:
     - name: Run fast Flax TPU tests
       if: ${{ matrix.config.framework == 'flax' }}
       run: |
+        apt-get update && apt-get install libsndfile1-dev libgl1 -y
```

Comment:
It's there because the underlying Docker container doesn't have the dependencies yet. It needs to be removed once the container has been pushed. 

For the rest, their Docker containers already have these deps either installed or the tests don't need them -- hence no failure.